src/styles/bloks/header.scss

```diff
@@ -0,0 +1,48 @@
+.header {
+  background-image: url(../images/photo-woman.png);
+  background-size: cover;
+  background-position: center;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    height: 100vh;
+  }
+
+  &__waves {
+    height: 57px;
+    width: 48px;
+    margin-bottom: 16px;
+    background-image: url(../images/Sound_waves.svg);
+    background-size: cover;
+
+    @include onTablet {
+      width: 64px;
+      height: 76px;
+      margin-bottom: 40px;
+    }
+  }
+
+  &__title {
+    margin: 0;
+    font-style: italic;
+    font-weight: 800;
+    font-size: 32px;
+    line-height: 100%;
+    letter-spacing: -1.5px;
+
```

Comment:
Add these properties to variables. 
Usually, the main power of scss is creating themes, so it's better to use static values as variables and unite such properties into mixins for page titles, page links, buttons, and so on.